non-critical-infra/packages/encrypt-email/encrypt-email.py

```diff
@@ -136,34 +130,31 @@ def login(address_id: str, force: bool) -> None:
 
     hashed_password = hash_password(password)
 
-    project_root = find_relative_project_root()
-    non_critical_infra_dir = project_root / "non-critical-infra"
-
     secret_path = non_critical_infra_dir / f"secrets/{address_id}-email-login.umbriel"
     encrypt_to_file(hashed_password, secret_path, force)
 
-    default_nix = non_critical_infra_dir / "modules/mailserver/default.nix"
-    assert default_nix.exists()
-
     nix_code = dedent(
         f"""\
         "{address_id}@nixos.org" = {{
           forwardTo = [
             # Add emails here
           ];
-          loginAccount.encryptedHashedPassword = ../../secrets/test-sender-email-login.umbriel;
+          loginAccount.encryptedHashedPassword = ../../secrets/{address_id}-email-login.umbriel;
         }};
         """
     )
     click.secho()
     click.secho("Now add this login account to ", nl=False)
-    click.secho(default_nix, fg="blue", nl=False)
-    click.secho(". Search for '", nl=False)
-    click.secho("### Mailing lists go here ###", fg="blue", nl=False)
+    click.secho(mailing_lists_nix, fg="blue", nl=False)
     click.secho("'. Add or edit an entry that looks like this:")
     click.secho()
     click.secho(indent(nix_code, prefix=" " * 4), fg="blue")
 
 
 if __name__ == "__main__":
+    project_root = find_relative_project_root()
+    non_critical_infra_dir = project_root / "non-critical-infra"
+    mailing_lists_nix = non_critical_infra_dir / "modules/mailserver/mailing-lists.nix"
+    assert mailing_lists_nix.exists()
```

Comment:
FYI: I'm going to make these globals a little more obvious by putting them near the top of the file and capitalizing them.

Reply:
Oh, I apparently don't have power to push to your branch.

I'll merge this and make the change in a followup PR.

Reply:
You now should be able to push to the branch.

Reply:
Thank you!